components/registry-facade/pkg/registry/manifest.go

```diff
@@ -219,7 +219,6 @@ func (mh *manifestHandler) getManifest(w http.ResponseWriter, r *http.Request) {
 		}
 
 		dgst := digest.FromBytes(p).String()
-		span.LogKV("manifest", string(p))
```

Comment:
Why remove this? 
it's been rather handy at times

Reply:
> it's been rather handy at times

Do we need that in the tracing? Can we replace that with a debug log?